docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -93,10 +93,65 @@ PIDFile=/run/user/1000/overlay-containers/ccfd5c71a088768774ca7bd05888d55cc28769
 [Install]
 WantedBy=multi-user.target default.target
 ```
+The generated .service file can be used to control the status of the pod and the underlying containers once they are enabled.
 
+Create systemd unit files for a pod. Since systemctl defaults to using the root user all the changes using the systemctl can be seen by appending sudo to the podman cli commands.
+
+```
+$ sudo podman pod create --name systemd-pod
+272d2813c7988ff26ef3d9386a113e001604150442d1184de74d31d6f02d6bb1
+$ sudo podman create --pod systemd-pod ngix top
+927aadd2d18ece4adbdf8037866d8e146f4f7bbcd7b47029ef27df722195f60a
+$ sudo podman generate systemd --files --name systemd-pod
+/home/user/pod-systemd-pod.service
+/home/user/container-awesome_banach.service
+$ sudo cp pod-systemd-pod.service container-awesome_banach.service /usr/lib/systemd/user
+$ systemctl enable pod-systemd-pod.service
+$ sudo podman pod ps
+POD ID         NAME          STATUS    CREATED          # OF CONTAINERS   INFRA ID
+272d2813c798   systemd-pod   Created   29 minutes ago   2                 6c5d116f4bbe
+$ systemctl start pod-systemd-pod.service
+$ sudo podman pod ps
+POD ID         NAME          STATUS    CREATED          # OF CONTAINERS   INFRA ID
+272d2813c798   systemd-pod   Running   29 minutes ago   2                 6c5d116f4bbe
+$ podman pod ps # 0 Number of pods on user thread.
+$ systemctl stop pod-systemd-pod.service
+$ sudo podman pod ps
+POD ID         NAME          STATUS   CREATED          # OF CONTAINERS   INFRA ID
+272d2813c798   systemd-pod   Exited   29 minutes ago   2                 6c5d116f4bbe
+```
+
+Create systemd unit files for a pod with one simple alpine containers. And Use the generated service file to modify the pod status as a non root uuser. Note: We are not using 'sudo' with our podman commands hence to do any modifications using the .service the systemctl should point to the current user be using --user flag.
```

Comment:
"alpine containers" -> "alpine container"

Reply:
I'd drop "And" and just stick with "Use" to start the second sentence.  If you keep the "And", lower case the "Use".

Reply:
Suggest:  "Note: We are not using 'sudo' with our Podman commands, hence to do any modification using the .service file the systemctl should point to the current user by using the --user flag."

Reply:
Fixed the old typos. @TomSweeneyRedHat PTAL now.